src/Stepper/Step.tsx

```diff
@@ -0,0 +1,112 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import * as _ from "lodash";
+
+import {Button} from "../Button/Button";
+
+import "./Step.less";
+
+const propTypes = {
+  className: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  showTitle: PropTypes.bool,
+  showMessage: PropTypes.bool,
+  showDescription: PropTypes.bool,
+  index: PropTypes.number,
```

Comment:
let's not make this an index - recommend calling it `id` instead and making it a string - that way, users aren't limited to numbers and i could, for example, use semantic IDs like `"intro"`, `"emailSettings"`, etc

Reply:
the step number (index) is what is rendered in the step icon. Long semantics ID's could not be used, but possibly steps like "A", "B", "C"?

Reply:
ohh, i think those should be separate, conceptually and semantically.

i think baking the notion of an "index" into the `Step` might be a bit much, since a `Step` doesn't really care or need to know where it falls in the order. it's more of a `label` than an `index` (and i agree, i could imagine a case where we label them `"A"`, `"B"`, `"C"` instead. the `Stepper`, on the other hand, can take care of ordering/orchestration and deciding what label is rendered in each `Step`

i think the `id` on the other hand should be required and should identify each step uniquely (and that's what i would include instead as the argument of the `onClick()`)